org.eclipse.winery.repository/src/test/java/org/eclipse/winery/repository/export/CsarExporterTest.java

```diff
@@ -99,7 +108,7 @@ public void csarDoesNotContainHashes() throws Exception {
     @Test
     public void testCsarFilesAreMentionedInTheManifest() throws Exception {
         Map<String, Object> exportConfiguration = new HashMap<>();
-        exportConfiguration.put(INCLUDE_HASHES.toString(), true);
+        exportConfiguration.put(INCLUDE_HASHES.name(), null);
```

Comment:
Why do you put `null`?

Reply:
I have changed the semantics of `exportConfiguration`. Now, only the existence of an entry with the key "X" means that the option "X" is chosen.
The value of the entry with the key "X" is an additional object that might by needed for the feature "X".
In the case of `INCLUDE_HASHES` no additional object is required. But for `STORE_IMMUTABLY` a `HashMap` is required to store a list of all files that need to be stored in the immutable storage.
It is convenient to put the `HashMap` here since the configuration object is already being passed around to all relevant methods. Otherwise, I would need to pass this `HashMap` separately to all of these methods, a solution that does not scale well if other configuration entries require similar treatment, or I would make the `HashMap` globally accessible _somehow_ which I also don't like.

Reply:
The `HashMap` is now unnecessary because all files are stored in the `refMap` `HashMap`.
However, the value here is still null, as using any other value (e.g., `true`) is misleading as it will not be used (we only care for the existence of the corresponding key).
Specific keys have some values which are used (see the class `CsarExportOptions`)